core/src/main/scala/cromwell/core/PathCopier.scala

```diff
@@ -23,7 +24,12 @@ object PathCopier {
     */
   def copy(sourceFilePath: Path, destinationFilePath: Path): Unit = {
     Option(File(destinationFilePath).parent).foreach(_.createDirectories())
-    File(sourceFilePath).copyTo(destinationFilePath, overwrite = true)
-    ()
+    try {
+      File(sourceFilePath).copyTo(destinationFilePath, overwrite = true)
+      ()
+    } catch {
```

Comment:
Is there a reason this can't convert the return type of `def copy` to `Try[Unit]` instead? I don't trust us to pay attention to what exceptions might be thrown by methods otherwise.